op-node/metrics/metrics.go

```diff
@@ -36,6 +36,10 @@ const (
 	BatchMethod = "<batch>"
 )
 
+type Bucketable interface {
+	PlaceIntoBucket(*prometheus.GaugeVec)
+}
+
```

Comment:
See other comment, this looks fancy but imo removes the abstraction that the `metrics` package should provide, an abstraction around prometheus.